pkg/platform/apiserver/versionproxy.go

```diff
@@ -0,0 +1,51 @@
+package apiserver
+
```

Comment:
1. please place and rename this file to tke/pkg/platform/proxy/version/rest/storage.go
2. add some comments to highlight